docs/tutorials/intro.md

```diff
@@ -156,19 +169,19 @@ use icu::calendar::DateTime;
 use icu::datetime::{DateTimeFormatter, options::length};
 
 fn main() {
-    let date = DateTime::try_new_gregorian_datetime(2020, 10, 14, 13, 21, 28)
-        .expect("Failed to create a datetime.");
-
     let options = length::Bag::from_date_time_style(length::Date::Long, length::Time::Medium);
 
+    let dtf = DateTimeFormatter::try_new_with_any_provider(&icu_testdata::any(), &locale!("ja").into(), options.into())
+        .expect("Failed to initialize DateTimeFormatter");
+
+
+    let date = DateTime::try_new_gregorian_datetime(2020, 10, 14, 13, 21, 28)
```

Comment:
Big Issue: Never use `try_new_gregorian_datetime` and then pass the result to a `DateTimeFormatter` generated from a variable locale. I continue to fear that this is going to be one of the biggest footguns in our ICU4X API decisions. It definitely doesn't belong in our tutorial!